contrib/babelfishpg_tsql/src/dbcmds.c

```diff
@@ -88,12 +88,14 @@ gen_createdb_subcmds(const char *dbname, const char *owner)
 	const char     *db_owner;
 	const char     *guest;
 	const char     *guest_schema;
+	Oid       	owner_oid;
 
 	schema = get_dbo_schema_name(dbname);
 	dbo = get_dbo_role_name(dbname);
 	db_owner = get_db_owner_name(dbname);
 	guest = get_guest_role_name(dbname);
 	guest_schema = get_guest_schema_name(dbname);
+	owner_oid = get_role_oid(owner, true);
```

Comment:
can be `bool role_is_sa = role_is_sa(get_role_oid(owner, true))` and use `role_is_sa` at other places.

Reply:
yes done